mhapsite/mhap/views.py

```diff
@@ -205,12 +205,13 @@ def index(request):
 
 
 
-    #quote=None
+    quote=None
     try:
         second_quote = Quote.objects.get(id=2)
+        quote=second_quote
```

Comment:
Everything looks syntactically correct but I don't understand the purpose of these changes in views.py.